### PR TITLE
Nc/cleaning

### PR DESCRIPTION
**Pull request by nc6**

### CODE REVIEW

File: nix/default.nix [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+let
+  haskell = import (./haskell.nix-src.nix) { inherit pkgs; };
+
+  pkgSet = haskell.mkStackPkgSet {
+    stack-pkgs = import ./pkgs.nix;
+    pkg-def-extras = [];
+    modules = [
+      {
+        # The SQLite tests break due to not finding 'echo'. We disable them for now.
+        packages.funflow.components.tests.unit-tests.testFlags = ["-p '! /SQLite/'"];
```
Comment (aherrmann) [inline nix/default.nix]: Maybe a missing dependency on `coreutils`?

File: nix/regenerate.sh [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+nix build -f https://github.com/input-output-hk/haskell.nix/archive/master.tar.gz nix-tools --out-link nt
```
Comment (aherrmann) [inline nix/regenerate.sh]: It's not clear to me which script I need to call to update the nix definitions (`regenerate.sh`, or `update-nixpkgs.sh`, or both) and which files under `nix/` are generated, which are modified, and which are handwritten. Might be worth 
 a few lines in the README.

Comment (nc6) [reply nix/regenerate.sh]: Yes, good point! I really should be better at documentation, given the circumstances...

File: nix/default.nix (hunk 1 above)
Comment (nc6) [reply nix/default.nix]: Right, but also something about how/where we're searching for binaries.

Comment (nc6) [reply nix/default.nix]: Specifically: simply adding `coreutils/bin` to the path didn't actually fix this problem!

**Pull request merged**